src/main/java/org/codehaus/mojo/license/AbstractAddThirdPartyMojo.java

```diff
@@ -1004,7 +1010,15 @@ void writeThirdPartyFile()
             {
                 licenseMap1 = licenseMap.toLicenseMapOrderByName();
             }
-            thirdPartyTool.writeThirdPartyFile( licenseMap1, thirdPartyFile, isVerbose(), getEncoding(), fileTemplate );
+            if ( createCSV )
+            {
+              getLog().info("Creating CSV file");
+              thirdPartyTool.writeThirdPartyFile( licenseMap1, thirdPartyFile, isVerbose(), getEncoding(), fileTemplate );
```

Comment:
Is it me, or is the only difference between the `if` and the `else` branch a log statement? Lines 1016 and 1020 seem identical..?

Reply:
+1